projects.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Projects
+---
+# Vac projects
+There are several projects under Vac. These are largely independent and can be seen as specific bets.
+
+
+## Waku Research
+
+Basically what Vac is primarily focused on today. Serves Waku Product, and aims to making Waku a self-sustained robust network with strong and modular privacy guarantees. This also includes secure messaging research and things like p2p/networking.
+
```

Comment:
As commented in hackmd, remove everything except first para here for now. We'll do greenfield content at a later stage. The hackmd this is based on was for private planning and not for public consumption at this stage